llvm/test/tools/llvm-reduce/thinlto-preserve-uselistorder.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: opt --thinlto-bc --thinlto-split-lto-unit %s -o %t.0
```

Comment:
I assume the change should affect %t.0 so that can be checked too?

Reply:
Not really relevant here, and redundant with the final result check